README.md

```diff
@@ -32,6 +32,16 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+
+## new Environment variables
+
+SECRET_KEY=secret-key
```

Comment:
Minor but it's probably worth explaining a little more about how to get these values and what they represent (for instance, it's not obvious that TRELLO_LISTID1 should be the list ID for your TO-DO column.